cli/crates/cli/src/cli_input.rs

```diff
@@ -17,9 +48,24 @@ pub struct DevCommand {
     /// Do not listen for schema changes and reload
     #[arg(long)]
     pub disable_watch: bool,
-    /// Log more events taking place such as introspection queries
-    #[arg(short, long)]
-    pub debug: bool,
+    /// Log level to print from function invocations, defaults to 'log-level'
+    #[arg(long)]
```

Comment:
I'd maybe name this since the parameter would be "FUNCTION_LOG_LEVEL"

Reply:
The others as well

Reply:
Sorry, not following. What parameter would be 'FUNCTION_LOG_LEVEL'?

Reply:
@yoav-lavi can you elaborate?

Reply:
Specifically referring to ` --log-level-functions <LOG_LEVEL_FUNCTIONS>`, `LOG_LEVEL_FUNCTIONS` as a parameter name seems a bit weirdly put